towhee/serve/triton/pipeline_client.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import numpy as np
+
+from towhee.utils.np_format import NumpyArrayDecoder, NumpyArrayEncoder
+from towhee.serve.triton.constant import PIPELINE_NAME
+
+
+class Client:
+    """
+    Triton HttpClient
+
+    Args:
+        url(`string`): IP address and port for the triton server, such as '<host>:<port>' and  '127.0.0.1:8001'.
+        model_name(`string`): Model name in triton, defaults to 'pipeline'.
+
+    Examples:
+        >>> from towhee import triton_client
+        >>> client = triton_client('<your-ip>:<your-port>')
+        >>> res = client('your-data')
+    """
+    def __init__(self, url: str, model_name: str = PIPELINE_NAME):
+        from towhee.utils.tritonclient_utils import httpclient  # pylint: disable=import-outside-toplevel
+        self._client = httpclient.InferenceServerClient(url)
+        self._model_name = model_name
+
+    def __call__(self, *args):
```

Comment:
The server will use the `batch` interface for all requests, in `__call__` function, we should unpack the data from the response.